lua/nvim-treesitter/parsers.lua

```diff
@@ -107,6 +107,14 @@ list.astro = {
   maintainers = { "@virchau13" },
 }
 
+list.authzed = {
+  install_info = {
+    url = "https://github.com/mleonidas/tree-sitter-authzed",
+    files = { "src/parser.c" },
+    branch = "main",
+  },
```

Comment:
Would you like to maintain it?

Reply:
I was originally going to leave it without a maintainer but since there are additional syntax features coming in the near future, sure, I’ll sign up to maintain this integration.

Reply:
We kinda do expect a maintainer, otherwise it'll bit rot ;)

Reply:
That's fair; I noticed some other entries didn't have one, so I took that as precedent :) 

I've indicated to the author of the tree-sitter grammar that I will maintain the nvim-treesitter integration so I am happy to make that official here too.

Reply:
> That's fair; I noticed some other entries didn't have one, so I took that as precedent :)

Those are invitations for someone else to pick them up ;)

Reply:
@clason speaking of falling out of date; the authzed tree-sitter project was refactored between when I opened this PR against nvim-treesitter and now. I was not sure whether to close this PR and open a new one, or get this PR merged first and then submit a new PR to update the highlights file. I was leaning toward the latter because I am learning a lot about the highlights query file (and y'all's preferences) in this PR that I will need to take back and submit as changes to the authzed tree sitter repo anyway.

Reply:
I think the best way then is to just mark this as draft and update when upstream is done. That will minimize breaking changes. (We rely on automation for parser updates, which require manual intervention on any breaking update.)